tests/js/setup-globals.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import intlLocalesSupported from 'intl-locales-supported';
+import intl from 'intl';
+import React from 'react';
+
+if ( global.Intl ) {
+	// Check for locales other than American English.
+	if ( ! intlLocalesSupported( [ 'en-US', 'de-DE' ] ) ) {
+		global.Intl.NumberFormat = intl.NumberFormat;
+		global.Intl.DateTimeFormat = intl.DateTimeFormat;
+	}
+} else {
+	throw new Error( 'Your version of node is very old and does not support `Intl`. Please use at least node 10.' );
```

Comment:
Is Node 10 required? It seems that `Intl` is mentioned as far back as [Node 8 in the docs ](https://nodejs.org/docs/latest-v8.x/api/intl.html#intl_internationalization_support) but seems to depend on how Node was built.

Reply:
I think 10 is safer, but it's also the version we specify in our `.nvmrc` anyway, so telling users to use it I think is fine.